src/color/border.json

```diff
@@ -51,33 +51,39 @@
       },
       "disabled": {
         "onLight": {
-          "value": "{color.base.disabled.value}",
+          "value": "{color.base.gray.200.value}",
           "public": true,
           "onLight": true,
           "usage": "Disabled border color on light backgrounds",
           "wcag": "n/a"
         },
         "onDark": {
-          "value": "{color.base.disabled.value}",
+          "value": "{color.base.gray.200.value}",
           "public": true,
           "onDark": true,
           "usage": "Disabled border color on dark background",
           "wcag": "n/a"
         }
       },
       "Focus": {
-        "onLight": {
+        "newState": {
```

Comment:
This is a whole new action-based color category ... need a new name/reason

Reply:
@corobrie this one still needs your attention.

Reply:
Need a bit more info on what this is trying to achieve (for my knowledge). Is this to denote a state of a component (active, focus, tapped, etc.)?

Reply:
In your previous [pr](https://github.com/AlaskaAirlines/OrionDesignTokens/pull/22/files#diff-7c681f482810613828e7b41450aa7dfaR69) you suggested `gray`. 

This update here, within the context of `focus` still has `onDark` and `onLight`. I just used the placeholder name of `newState` with this note to ask; what do we want to name this?